src/CMakeLists.txt

```diff
@@ -667,23 +661,13 @@ clang_tidy_check(MIOpen)
 
 function(target_internal_library TARGET)
     target_link_libraries(${TARGET} PRIVATE ${ARGN})
-    set(PASS_ARGS debug optimized)
-    set(DEPS)
-    foreach(DEP ${ARGN})
-        if(DEP IN_LIST PASS_ARGS)
-            list(APPEND DEPS ${DEP})
-        else()
-            list(APPEND DEPS $<BUILD_INTERFACE:${DEP}>)
-        endif()
-    endforeach()
-    target_link_libraries(${TARGET} INTERFACE ${DEPS})
```

Comment:
@pfultz2 Do we no longer need this patch for "debug" and "optimized"?

Reply:
Ping @pfultz2 and @JehandadKhan

Reply:
No, it isnt used anymore. I think this was used before we could use boost cmake targets.